iot_verifier/migrations/10_packet.sql

```diff
@@ -0,0 +1,7 @@
+create table packet (
+    id serial primary key,
```

Comment:
i am not sure but the max over a 7day period in feb amounted to 82m packets.  As we will be clearing these out ever reward period ( so max 24hours ) i32 likely gives plenty of headroom.  That said we dont loose much by defaulting to bigserial

Reply:
if you’re just deleting the table contents every 24 hours I believe that will _not_ reset the serial integer since Postgres keeps track of the `last_value` in a separate associate `packet_seq` table and increment that with every record generated. So you’ll need to drop and recreate both tables periodically or perform special SEQUENCE update queries to reset the serial number back to 1 each time you drop the packet table contents